Mathlib/Algebra/Ring/UnitEquivs.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Rodriguez, Johan Commelin
+-/
+import Mathlib.GroupTheory.GroupAction.ConjAct
+import Mathlib.RingTheory.Subring.Basic
```

Comment:
This import is only necessary because of `unitsCentralizerEquivSubring` and `centerUnitsEquivUnitsRingCenter` about which I have doubts.

If it turns out we don't need those lemmas then I think these results could go in the `GroupTheory.GroupAction.ConjAct` file.